### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by Ibtisam34**
- [ ]  In this project, I built a simple HTML list of To Do tasks. The list will be styled according to the specifications listed later in this lesson. This simple web page will be built using webpack and served by a webpack dev server.

### CODE REVIEW

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @Ibtisam34 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request :heavy_multiplication_x: 


Invalid


You have submitted a project, that has no linters checks set up. Kindly fix that and make sure all of the test cases are green before resubmiting . 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @Ibtisam34, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## ❌ Invalid Code Review Request

You have submitted a project, that does not have a descriptive readme. Kindly update your readme 👍🏽

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+const toDosDiv = document.getElementById('todo-list');
+const toDosLists = (myToDos) => <div class = "toDo-list">
+  <ul>
+   <input type="checkbox"></input>
+    <span>${myToDos.description}</span>
+  </ul>
+</div>;
+toDosDiv.innerHTML = `
+${toDos.map(toDosLists).join('')}`;
```
Comment (Olamarx) [inline src/index.js:12]: - [ ] Kindly populate the UI with the array of data you created. You can try to follow this format 

```javascript
const populateTasks = (task, parentElement) => {
  const { description, completed, index: id } = task;
  const innerHTML = `
         <p>${description}</p>
         <p>${completed ? 'completed' : 'not completed'}</p>`;
  const newTodoListElement = document.createElement('li');
  Object.assign(newTodoListElement, { innerHTML, id });
  parentElement.appendChild(newTodoListElement);
};
```

Comment (Olamarx) [inline src/index.js]: - [ ] Kindly populate the UI with the array of data you created. You will need to loop the array and you can try to follow this format after looping so as to render it.

```javascript
const populateTasks = (task, parentElement) => {
  const { description, completed, index: id } = task;
  const innerHTML = `
         <p>${description}</p>
         <p>${completed ? 'completed' : 'not completed'}</p>`;
  const newTodoListElement = document.createElement('li');
  Object.assign(newTodoListElement, { innerHTML, id });
  parentElement.appendChild(newTodoListElement);
};
```

File: README.md [hunk 2]
```diff
@@ -1 +1,45 @@
...
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+  Cindy Shin the author of the original design [https://www.behance.net/gallery/29845175/CC-Global-Summit-2015]
```
Comment (Olamarx) [inline README.md]: [OPTIONAL] Kindly remove this as I know you actually forgot.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @Ibtisam34, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- All linters passed ✔️ 
- Proper setup of webpack 👍🏼 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ibtisam34** [comment]
- [ ] @Olamarx hi sir kindly 🙏 could you check again the review  was the changes necessary  and if so help me how could I do it

**Comment by Ibtisam34** [comment]
Plz assis me with the code 🙏 😔 where do you want me to change  and to
replace  what

**Review by ichala — APPROVED** [review]
Hi @Ibtisam34 ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Webpack Installed✔️ 
- Correct Linters ✔️
- Clean Code ✔️



 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi @Ibtisam34 ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Webpack Installed✔️ 
- Correct Linters ✔️
- Clean Code ✔️



 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi @Ibtisam34 ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Webpack Installed✔️ 
- Correct Linters ✔️
- Clean Code ✔️



 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**